test.js

```diff
@@ -121,3 +121,30 @@ test('invalid encoding values', async function (t) {
 
   await seed.close()
 })
+
+test('put/get/del metadata', async function (t) {
+  t.plan(2)
+
+  const seed = new SeedBee(new Hypercore(RAM))
+  const key = 'key'
+  const value = '*'
+
+  await seed.metadata.put(key, value)
+  t.is(value, await seed.metadata.get(key))
+
+  await seed.metadata.del(key)
+  t.is(null, await seed.metadata.get(key))
+
+  await seed.close()
+})
+
+test('get empty metadata', async function (t) {
+  t.plan(1)
+
+  const seed = new SeedBee(new Hypercore(RAM))
+  const key = 'key'
+
+  t.is(null, await seed.metadata.get(key))
```

Comment:
I see you have another test where you're already testing put/get/del all together, if you move this metadata.get at the beginning of the test above then you have it all unified